src/components/sortableGridList/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {render} from '@testing-library/react-native';
+import View from '../../view';
+import Text from '../../text';
+import SortableGridList, {SortableGridListProps} from '../';
+import {useDraggableDriver} from '../../../testkit/new/useDraggable.driver';
+import {useComponentDriver} from '../../../testkit/new/Component.driver';
+
+const testID = 'sortable-grid-list';
+const TEST_DATA = Array(9)
+  .fill(0)
+  .map((_, index) => ({id: `${index}`}));
+const itemsTestId = (id: string) => `${testID}.item.${id}`;
+const testRenderItem: SortableGridListProps<(typeof TEST_DATA)[number]>['renderItem'] = ({item}) => {
+  return (
+    <View height={50} testID={itemsTestId(item.id)}>
+      <Text>{item.id}</Text>
+    </View>
+  );
+};
+
+const Testcase = (props: Omit<SortableGridListProps, 'data' | 'renderItem'>) => {
+  return <SortableGridList {...props} numColumns={3} testID={testID} data={TEST_DATA} renderItem={testRenderItem}/>;
+};
+
+describe('SortableGridlist', () => {
+  it('should render a sortable grid list', () => {
+    render(<Testcase/>);
+  });
+  it('should reorder by index', () => {
+    const onOrderChange = jest.fn();
+    const renderTree = render(<Testcase onOrderChange={onOrderChange} orderItemsByIndex/>);
+    const driverItem = useDraggableDriver(useComponentDriver({renderTree, testID: itemsTestId('0')}));
+    driverItem.drag(150); // Items height is 50 but dragging 100 doesn't work for some reason. 150 works and drags one row down.
```

Comment:
I'm not sure what the calculation for the drag is. I used 150 because it worked but I don't know If I can make a driver that will be correct for all cases.

Reply:
You can look at `SortableListItem.driver.new.ts`; even if it's not perfect, it's better than nothing

Reply:
In the `SortableListItem` we have a default height for the items so its easy to calculate. Also the widths are kind of calculated based on the props. I don't really think we need to invest much time in this currently tbh. We can come back to this some other time.

Reply:
You've added tests, so it seems logical to add a driver...
We will probably never have time to refactor this...

Reply:
We'll probably get to this if the component is used more

Reply:
How do you intent to measure usage?